doc/docs/master/concepts/advanced-concepts/globalID.md

```diff
@@ -164,32 +175,31 @@ Let's explain the origin of each commit.
 
 - Besides  the `USER` and `AUTO` commits, notice a set of `ALIAS` commits in `edges.spec` and `montage.spec`:
 ```shell
-    pachctl inspect commit edges.spec@336f02bdbbbb446e91ba27d2d2b516c6 --raw
+    $ pachctl inspect commit edges.spec@336f02bdbbbb446e91ba27d2d2b516c6 --raw
 ```
 The version of each pipeline within their respective `.spec` repos are neither the result of a user change, nor of an automatic change.
 They have, however, contributed to the creation of the previous `AUTO` commits. 
-To make sure that we have a complete view of all the data and pipeline versions involved in all commits resulting from the initial 
+To make sure that we have a complete view of all the data and pipeline versions involved in all the commits resulting from the initial 
 `put file`, their version is kept as `ALIAS` commits under the same global ID.
 
 For a fuller view of GlobalID in action, take a look at our [GlobalID illustration](https://github.com/pachyderm/pachyderm/tree/master/examples/globalID).
 
 ## Track Provenance Downstream
 
-Pachyderm provides the `wait commitset` command that enables you
+Pachyderm provides the `wait commit <commitID>` command that enables you
 to track your commits downstream as they are produced. 
 
-Unlike the `inspect commitset`, each line is printed as soon as a new commit of your set finishes.
+Unlike the `list commit <commitID>`, each line is printed as soon as a new (sub) commit of your global commit finishes.
 
-Change `commitset` in `jobset` to list the jobs related to your jobset as they finish processing a commit.
+Change `commit` in `job` to list the jobs related to your global job as they finish processing a commit.
 
-## Squash Commitset
+## Squash Commit
 
-`pachctl squash commitset 1035715e796f45caae7a1d3ffd1f93ca`
-combines all the file changes in the commits of a commitset 
-into their children and then removes the commitset.
+`pachctl squash commit 1035715e796f45caae7a1d3ffd1f93ca`
+**combines all the file changes in the commits of a global commit
+into their children** and then removes the global commit.
 This behavior is inspired by the squash option in git rebase.
 No data stored in PFS is removed.
 
 !!! Warning
-    Squashing a commitset with no children results in the head of the branch disagreement with the state of the world. 
-    So a new identical commitset will be created.
+    Squashing a global commit on the head of a branch (no children) results in a new identical global commit being created.
```

Comment:
I was wrong about this, and per some recent discussions, I think our squash commit behavior in this case is just broken (when you squash a (global) commit, any commits which are the head of their branches are effectively deleted, their data thrown away). Not sure what the state of it will be in 2.0 exactly, might make sense to just omit this for now